### ISSUE TITLE
Can't message people after account reset

### ISSUE THREAD

**Issue by marceljay** [keybase/keybase-issues#3551]
Probably related to issue #3351 

When trying to message someone with whom a chat existed before account reset, an error occurs. I agree, letting you back in requires confirmation, but it should be possible to establish a new chat with said person.

When trying to delete the chat completely it also spawns an error:
```
Error: ERROR CODE 2623 - Unable to decrypt chat message (transient): You are not a member of this team (error 2623) in method chat.1.local.postDeleteHistoryByAge at new s [...]
```

**Comment by plttn** [comment]
The short version is there has to be no way of getting around the lock on a reset account. From the other party's perspective, there's no way to tell if the account reset was intentional by you, or by an attacker. Until they can verify that this was an intended reset by you, any way to chat them again wouldn't be safe.

**Comment by marceljay** [comment]
I must oppose your view. On an account that was reset I can still message new people that I never messaged with potentially malicious content. Yet they don't know if they can trust me, so it's up to them if they block me. Former contacts can just be notified about the account reset and that they need to be careful. This is also how Signal Messenger handles new phones/keys. You get notified the device changed, then you can ask e.g. for social proof.  I think the current UX is really suboptimal.

**Comment by plttn** [comment]
>On an account that was reset I can still message new people that I never messaged with potentially malicious content

Right, but they didn't have an existing relationship with you, so there's no harm in the fact that your account status has changed.

**Comment by marceljay** [comment]
So you would argue, that harm is done to them because they could ignore a warning message stating my account could be compromised? Isn't that a bit patronizing? 
Do you then also suggest send them an email or call them to tell them: "hey, please accept my new account and let me send you a message" ?

**Comment by plttn** [comment]
To be clear, are you referring to a new chat or an old chat? 

If it's a new chat, there's nothing inherently dangerous, because there's no prior chat and expectation of account-controlling person mapping.

If it's an old chat however, the onus is on the other party to decide "is this a legit reset of theirs, let me check their proofs and see whats going on".

**Comment by jimbobhickville** [comment]
I understand the reasoning behind requiring re-authorization, and I agree with it, but it's really frustrating that you can't send them a message via keybase and then they get prompted "this account was reset, do you want to trust it?" or something. You have no means of notifying them that you're back online unless you have another method of contacting them. 

Or even just some way to clear the old chat and start a new chat with an existing contact. I don't care about the message history, just want to be able to chat some old contacts after having to reset my account.
